corona-warn-app/cwa-app-android
# Show additional high risk notifications (EXPOSUREAPP-12064)

### Description

New notifications for additional high risks. When the user clicks on the notification, or the next time the app is opened, a new dialog should be displayed: [screenshot of new dialog](https://user-images.githubusercontent.com/10398034/157615373-e9767765-2f7d-4051-8f62-6007a7a3db11.png). Once the user clicks "OK" on the dialog, it won't show up on subsequent app opens. 

Covers all subtasks of this story: https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12064

### How to test
- Set device time to real time
- Set radio box `INCREASED_RISK_DEFAULT` in Test Menu
- Click `CALCULATE RISK LEVEL`
- User should now get to the "initial high risk" state

Then there are 2 options to continue

_Option 1:_
- Leave the app in this state over night
- A risk calculation in the background (with an additional high risk) on the next day should trigger a new notification.

_Option 2:_
- Set the device time to one day in the future
- Open the app
- The risk calculation will be triggered, resulting in an "additional high risk"
- The dialog should be shown on the HomeFragment
- A notification should be triggered
